script/setup/hcsshim-version

```diff
@@ -0,0 +1 @@
+v0.10.0
```

Comment:
Not sure if a newline will break things here, but if not, perhaps we can add one (easier to work with). But that's just a "nice to have".

Reply:
It doesn't, changing!